ui/package.json

```diff
@@ -50,7 +50,7 @@
     "@typescript-eslint/parser": "^7.1.1",
     "@vitejs/plugin-react": "^4.2.1",
     "autoprefixer": "^10.4.19",
-    "eslint": "^9.0.0",
+    "eslint": "^8",
```

Comment:
Just a style question: Should this be `"^8.0.0"`, similar to like in line 38? Or does that even have a different meaning?

Reply:
`"^8"` and `"^8.0.0"` have identical meaning AFAIK - reference [here](https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept). But now that I think it through again, specifying minimum of the currently latest version `8.x.x` is probably more accurate, as it hasn't been tested with older versions.